tests/benchmark_test.py

```diff
@@ -214,3 +214,11 @@ def test_functions_call_error_above(fun):
     with pytest.raises(ValueError):
         fun(above_bounds)
 
+@pytest.mark.parametrize("fun",functions[:3])
```

Comment:
Why is this list being sliced to only the first 3 functions?

Reply:
That is because it shouldn't. That was done to make sure the test worked and then wasn't removed.